pkg/roachpb/span_config.proto

```diff
@@ -164,6 +164,14 @@ message SpanConfig {
   // RangefeedEnabled determines whether rangefeeds are enabled over the
   // specific range.
   bool rangefeed_enabled = 10;
+
+  // ExcludeDataFromBackup specifies if the table's row data has been marked to
```

Comment:
[mega nit] Reword this slightly to talk primarily in term of ranges/keyspaces instead, and only secondarily in terms of rows/tables. Talking about ExportRequests is fine since that's already part of the KV API.